cmake/export.cmake

```diff
@@ -1,36 +1,44 @@
-# TODO export package
-
-#include(GNUInstallDirs)
-
-# install(FILES src/dotprod.h DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/dotprod)
-
-# install(EXPORT dotprod-targets
-#     FILE dotprod-targets.cmake
-#     NAMESPACE Dotprod::
-#     DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake
-#     )
+# this registers the build-tree with a global CMake-registry
+export(PACKAGE cpp_bindgen)
 
 include(CMakePackageConfigHelpers)
 
-write_basic_package_version_file(cpp_bindgen-config-version.cmake
+# for build tree
+set(cpp_bindgen_MODULE_PATH ${PROJECT_BINARY_DIR}${CMAKE_FILES_DIRECTORY}/build-install/lib/cmake)
+set(cpp_bindgen_SOURCES_PATH ${CMAKE_SOURCE_DIR}/src)
```

Comment:
If c-bindings is added with add_subdirectory, will `CMAKE_SOURCE_DIR`, I would assume that `CMAKE_SOURCE_DIR` will point to the root source dir of the parent project? I have the feeling, this should be `PROJECT_SOURCE_DIR`.

Reply:
same also at other places

Reply:
yes, you are right! thanks.